src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * <p>
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package org.fundacion.jala.converter.models.parameter;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class AudioParameter extends Parameter {
+
+    private String format;
+    private String bitrate;
+    private String hz;
+    private String volume;
+    private String audioChannels;
+
+    public AudioParameter() {
+        super();
+    }
+
+    public AudioParameter(final String newFilePath) {
+        super(newFilePath);
+    }
+
+    public AudioParameter(final String newFilePath, final String newFormat, final String newBitrate, final String newHz, final String newVolume, final String newAudioChannels) {
+        super(newFilePath);
+        this.format = newFormat;
+        this.bitrate = newBitrate;
+        this.hz = newHz;
+        this.volume = newVolume;
+        this.audioChannels = newAudioChannels;
+    }
+
+    /**
```

Comment:
The description of all methods in this class is missing

Reply:
ok, i will correct it

Reply:
yes, I think that is missing description in the methods

Reply:
I don't understand, what is missing